tests/composition/test_autodiffcomposition.py

```diff
@@ -62,6 +62,38 @@ def test_report_prefs(self):
         # comp = AutodiffComposition()
         # assert comp.patience == 10
 
+class TestNonTrainingFunctionality:
+    @pytest.mark.parametrize("mode", ['Python',
+                                      pytest.param('LLVMRun', marks=pytest.mark.llvm),
+                                     ])
```

Comment:
this should be marked pytorch. otherwise, it will fail without the `torch` module